vg/vgimg/vgimg.go

```diff
@@ -20,8 +20,26 @@ import (
 	"golang.org/x/image/tiff"
 
 	"gonum.org/v1/plot/vg"
+	vgdraw "gonum.org/v1/plot/vg/draw"
 )
 
+func init() {
+	pf := func(w, h vg.Length) vg.CanvasWriterTo {
+		return PngCanvas{Canvas: New(w, h)}
+	}
+	vgdraw.RegisterFormat("png", pf)
+	jf := func(w, h vg.Length) vg.CanvasWriterTo {
+		return JpegCanvas{Canvas: New(w, h)}
+	}
+	vgdraw.RegisterFormat("jpg", jf)
+	vgdraw.RegisterFormat("jpeg", jf)
+	tf := func(w, h vg.Length) vg.CanvasWriterTo {
+		return TiffCanvas{Canvas: New(w, h)}
+	}
+	vgdraw.RegisterFormat("tif", tf)
+	vgdraw.RegisterFormat("tiff", tf)
```

Comment:
Can we split this into meaningful stanzas (one for each `func`).

Reply:
done.

Reply:
I was unclear, sorry, but this if fine. I meant jut adding blank lines between specific groups of registrations.